c.html.markdown

```diff
@@ -130,7 +130,9 @@ int main(void) {
   // can be declared as well. The size of such an array need not be a compile
   // time constant:
   printf("Enter the array size: "); // ask the user for an array size
-  char buf[0x100];
+  int size;
+  scanf("%d", &size);
```

Comment:
Thank you for the reference. fgets takes string, and sscanf can not take input from stdin. Though I stumbled upon fscanf ((fgets + sscanf)? lol) and I think its the best replacement.